libraries/classes/ResponseRenderer.php

```diff
@@ -506,4 +509,22 @@ public function callExit(string $message = ''): never
 
         exit;
     }
+
+    /**
+     * @psalm-param non-empty-string $url
+     * @psalm-param StatusCodeInterface::STATUS_*|null $statusCode
+     */
+    public function redirect(string $url, int|null $statusCode = null): void
```

Comment:
Instead of `$statusCode ?? StatusCodeInterface::STATUS_FOUND` 
```suggestion
    public function redirect(string $url, int|null $statusCode = StatusCodeInterface::STATUS_FOUND): void
```